### PR TITLE
[TASK] Update hints of Extension Builder

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See TYPO3-Documentation/T3DocTeam#174 for details.

### CODE REVIEW

File: Documentation/1-Installation/Index.rst [hunk 1]
```diff
@@ -1,59 +1,16 @@
...
-
-Then you can use the *Extension Manager* to activate the Extension Builder.
+To follow this documentation you need to have
 
+1. :doc:`TYPO3 installed and running <t3start:Installation/Index>`,
+2. the system extensions *Extbase* and *Fluid* and
+3. the `Extension Builder <https://extensions.typo3.org/extension/extension_builder>`__ extension installed.
 
+The Extension Builder helps you to create and manage your Extbase and Fluid
+based TYPO3 extensions by providing you a GUI.
```
Comment (susannemoog) [inline Documentation/1-Installation/Index.rst:14]: providing you a GUI sounds wrong.

File: Documentation/1-Installation/Index.rst [hunk 2]
```diff
@@ -1,59 +1,16 @@
...
+But note that you should still familiarize yourself with the generated files
```
Comment (susannemoog) [inline Documentation/1-Installation/Index.rst:15]: A sentence starting with 'But' sounds like a fragment.

**Review by susannemoog — CHANGES_REQUESTED** [review]

File: Documentation/1-Installation/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/1-Installation/Index.rst:14]: No problem. Adjust it to your liking so it sounds right to you. I will take care of the backport.

File: Documentation/1-Installation/Index.rst (hunk 2 above)
Comment (alexander-nitsche) [reply Documentation/1-Installation/Index.rst:15]: No problem. Adjust it to your liking so it sounds right to you. I will take care of the backport.

File: Documentation/1-Installation/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/1-Installation/Index.rst]: Adjusted by Tom.

File: Documentation/1-Installation/Index.rst (hunk 2 above)
Comment (alexander-nitsche) [reply Documentation/1-Installation/Index.rst]: Adjusted by Tom.

**Review by susannemoog — APPROVED** [review]

**Pull request merged**